test/unit/services/JobRoleServiceTest.ts

```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+import { JobRoleResponse } from "../../../src/models/JobRoleResponse";
+import { getAllOpenJobRoles, URL } from '../../../src/services/JobRoleService';
+import { expect } from 'chai';
+
+const jobRoleResponse: JobRoleResponse = {
+    roleId: 1,
+    roleName: "Software Developer",
+    location: "Toronto, CA",
+    capability: "Digital Services",
+    band: "Trainee",
+    closingDate: "2024-08-15 23:59:59",
+    roleStatus: 1
+}
+
+const mock = new MockAdapter(axios);
+
+describe('JobRoleService', function () {
+    describe('getAllOpenJobRoles', function () {
```

Comment:
should we have a test to check empty list?

Reply:
Yes for sure, just created one